spring-cloud-azure-samples/eventhubs-binder-sample/README.adoc

```diff
@@ -162,14 +188,14 @@ $ curl -X POST http://localhost:8080/messages?message=hello
 or when the app runs on App Service
```

Comment:
Better address the VM case as well.  Consider this `runs on App Service or VM`.

Reply:
done